```
ImportError while loading conftest '/__w/numpy/numpy/build-install/usr/lib/python3.10/site-packages/numpy/conftest.py'.
numpy/__init__.py:135: in <module>
    raise ImportError(msg) from e
E   ImportError: Error importing numpy: you should not try to import numpy from
E           its source directory; please exit the numpy source tree, and relaunch
E           your python interpreter from there.
```ImportError while loading conftest '/__w/numpy/numpy/build-install/usr/lib/python3.10/site-packages/numpy/conftest.py'.
numpy/__init__.py:135: in <module>
    raise ImportError(msg) from e
E   ImportError: Error importing numpy: you should not try to import numpy from
E           its source directory; please exit the numpy source tree, and relaunch
E           your python interpreter from there.

--- a/numpy/lib/tests/test_io.py
+++ b/numpy/lib/tests/test_io.py
@@ -443,7 +443,7 @@ class TestSaveTxt:
         
         # Native
         newline = os.linesep.encode()
-        np.savetxt(c, a, fmt='%1d', newline=None)
+        np.savetxt(c, a, fmt='%d', newline=None)
         c.seek(0)
         assert_equal(c.readlines(), [b'1 2'+newline, b'3 4'+newline],
                      err_msg='Native newline')
@@ -451,7 +451,7 @@ class TestSaveTxt:
         # POSIX
         newline = b'\n'
         c = BytesIO()
-        np.savetxt(c, a, fmt='%1d', newline=newline)
+        np.savetxt(c, a, fmt='%d', newline=newline)
         c.seek(0)
         lines = c.readlines()
         assert_equal(lines, [b'1 2'+newline, b'3 4'+newline],
@@ -460,7 +460,7 @@ class TestSaveTxt:
         # NT
         newline = b'\r\n'
         c = BytesIO()
-        np.savetxt(c, a, fmt='%1d', newline=newline)
+        np.savetxt(c, a, fmt='%d', newline=newline)
         c.seek(0)
         lines = c.readlines()
         assert_equal(lines, [b'1 2'+newline, b'3 4'+newline],
@@ -469,7 +469,7 @@ class TestSaveTxt:
         # Tab
         newline = b'\t'
         c = BytesIO()
-        np.savetxt(c, a, fmt='%1d', newline=newline)
+        np.savetxt(c, a, fmt='%d', newline=newline)
         c.seek(0)
         lines = c.readlines()
         assert_equal(lines, [b'1 2'+newline+b'3 4'+newline, ],

--- a/numpy/lib/tests/test_io.py
+++ b/numpy/lib/tests/test_io.py
@@ -446,7 +446,7 @@ class TestSaveTxt:
         np.savetxt(c, a, fmt='%1d', newline=None)
         c.seek(0)
         assert_equal(c.readlines(), [b'1 2'+newline, b'3 4'+newline],
-                     err_msg='Native newline)
+                     err_msg='Native newline')
 
         # POSIX
         newline = b'\n'
@@ -455,7 +455,7 @@ class TestSaveTxt:
         c.seek(0)
         lines = c.readlines()
         assert_equal(lines, [b'1 2'+newline, b'3 4'+newline],
-                     err_msg='POSIX newline)
+                     err_msg='POSIX newline')
 
         # NT
         newline = b'\r\n'
@@ -464,7 +464,7 @@ class TestSaveTxt:
         c.seek(0)
         lines = c.readlines()
         assert_equal(lines, [b'1 2'+newline, b'3 4'+newline],
-                     err_msg='NT newline)
+                     err_msg='NT newline')
 
         # Tab
         newline = b'\t'
@@ -473,7 +473,7 @@ class TestSaveTxt:
         c.seek(0)
         lines = c.readlines()
         assert_equal(lines, [b'1 2'+newline+b'3 4'+newline, ],
-                     err_msg='Tab newline)
+                     err_msg='Tab newline')
     
     def test_header_footer(self):
         # Test the functionality of the header and footer keyword argument.

--- a/numpy/lib/tests/test_io.py
+++ b/numpy/lib/tests/test_io.py
@@ -443,32 +443,37 @@ class TestSaveTxt:
         
         # Native
         newline = os.linesep.encode()
-        np.savetxt(c, a, newline=None, fmt='%1d')
+        np.savetxt(c, a, fmt='%1d', newline=None)
         c.seek(0)
-        assert_equal(c.readlines(), [b'1 2'+newline, b'3 4'+newline])
+        assert_equal(c.readlines(), [b'1 2'+newline, b'3 4'+newline],
+                     err_msg='Native newline)
 
         # POSIX
         newline = b'\n'
         c = BytesIO()
-        np.savetxt(c, a, fmt='%1d')
+        np.savetxt(c, a, fmt='%1d', newline=newline)
         c.seek(0)
         lines = c.readlines()
-        assert_equal(lines, [b'1 2'+newline, b'3 4'+newline])
+        assert_equal(lines, [b'1 2'+newline, b'3 4'+newline],
+                     err_msg='POSIX newline)
 
         # NT
         newline = b'\r\n'
         c = BytesIO()
-        np.savetxt(c, a, fmt='%1d')
+        np.savetxt(c, a, fmt='%1d', newline=newline)
         c.seek(0)
         lines = c.readlines()
-        assert_equal(lines, [b'1 2'+newline, b'3 4'+newline])
+        assert_equal(lines, [b'1 2'+newline, b'3 4'+newline],
+                     err_msg='NT newline)
 
         # Tab
+        newline = b'\t'
         c = BytesIO()
-        np.savetxt(c, a, fmt='%1d')
+        np.savetxt(c, a, fmt='%1d', newline=newline)
         c.seek(0)
         lines = c.readlines()
-        assert_equal(lines, [b'1 2'+newline, b'3 4'+newline])
+        assert_equal(lines, [b'1 2'+newline+b'3 4'+newline, ],
+                     err_msg='Tab newline)
     
     def test_header_footer(self):
         # Test the functionality of the header and footer keyword argument.

